pkg/vtpm/src/server.cpp

```diff
@@ -96,8 +100,16 @@ static int
 sendResponse (int sock, eve_tools::EveTPMResponse &response)
 {
     int rc = success;
+    if (response.ByteSize() > (MAX_PAYLOAD_SIZE - CODED_STRM_HDR_LEN)) {
+        cerr << "Response size " << response.ByteSize() << " is too large (maybe adjust MAX_PAYLOAD_SIZE)" << std::endl;
+        return failure;
+    }
     google::protobuf::uint32 size = response.ByteSize() + CODED_STRM_HDR_LEN;
-    char *resp_buffer = new char [size];
+    char *resp_buffer = new(nothrow) char [size];
```

Comment:
is it `std::nothrow` we are using? And why do we need nothrow in this context. Also we can use something like 
`std::unique_ptr<T> p = new(std::nothrow) T();` that way we can forget about removing resources in the end of the function

Reply:
`nothrow` becuse `handleRequest` should return an error, and we don't care **why** allocation failed, IMHO adding try/catch and then returning the error is just extra text in the code.

Reply:
using `std::unique_ptr` then I have extract the raw pointer when I want to use it :-/

Reply:
We can leave it as it is, because most of questions are related to implementation of this specific function, which is not part of this PR :)